internal/c/libqb.cpp

```diff
@@ -37148,6 +37148,26 @@ int32 func__resize() {
     return 0;
 }
 
+void sub__windowscale(int32 amount) {

```

Comment:
How is this intended to play with the `_Resize` option? Since the scaling respects the aspect ratio I think we could keep the current setting in place of either `_Smooth`, `_Stretch`, or off. Setting the `ScreenResizeScale` value appears to wipe it out.

Also, should the scaling be maintained if a new `SCREEN` is put in place? I don't really know, but it seems like a problem if it doesn't because if you set the screen to a new image to change the size, it will resize the window to the unscaled size until you do another `_WINDOWSCALE` call. That resizing back to the small window might be visible to the user.